controller/golden_files_test.go

```diff
@@ -199,7 +199,7 @@ func replaceTimes(str string) (string, error) {
 	hour = "([01][0-9]|2[0-3])"
 	minute = "([0-5][0-9])"
 	second = "([0-5][0-9]|60)"
-	tz := "(GMT|CEST|UTC)"
+	tz := "(GMT|CEST|UTC|IST)"
```

Comment:
Maybe we should instead make this a `tz := "(GMT|CEST|UTC|IST|[A-Z]+)"` so that we don't have to fix it the next time somebody from yet another different timezone runs the tests there are errors. I know that the regex could be simplified to be just `tz := "([A-Z]+)"` but I would like to keep the **idea** of what it represents by keeping the `GMT`s etc. inside.